fem/complex_fem.cpp

```diff
@@ -216,6 +226,19 @@ ComplexLinearForm::operator()(const ComplexGridFunction &gf) const
                           (*lfr)(gf.imag()) + s * (*lfi)(gf.real()));
 }
 
+bool SesquilinearForm::RealInteg()
+{
+   int nint = blfr->GetFBFI()->Size() + blfr->GetDBFI()->Size() +
+              blfr->GetBBFI()->Size() + blfr->GetBFBFI()->Size();
+   return !(nint == 0);
```

Comment:
It would be more concise to use (nint != 0).

Reply:
Done!